pages/events.jsx

```diff
@@ -79,7 +79,7 @@ var EventsPage = React.createClass({
           <section>
             <IconLinks>
               <IconLink
-                linkTo="fixme"
+                linkTo="/events/resources"
```

Comment:
This doesn't work like a standard link.

The linkTo attribute is a custom thing that allows a link to transition the menu colour. It's doesn't actually link, which is why the standard url won't work. You instead give it an id to the page.

To see the ids that are set, view the https://github.com/ScottDowne/teach.webmaker.org/blob/resources/lib/routes.jsx#L26 page.

That's the file from my pull request and the master branch.

You can see I setup a route called `name="event-resources"` so in order to link to it, use `event-resources` in the linkTo attribute.